relay-general/src/store/normalize/span/tag_extraction.rs

```diff
@@ -127,33 +140,44 @@ fn extract_shared_tags(event: &Event) -> BTreeMap<SpanTagKey, String> {
         if let Some(transaction_method) = transaction_method_from_request.or_else(|| {
             http_method_from_transaction_name(transaction_name).map(|m| m.to_uppercase())
         }) {
-            tags.insert(SpanTagKey::TransactionMethod, transaction_method);
+            tags.insert(SpanTagKey::TransactionMethod, transaction_method.into());
         }
     }
 
     if let Some(trace_context) = event.context::<TraceContext>() {
         if let Some(op) = extract_transaction_op(trace_context) {
-            tags.insert(SpanTagKey::TransactionOp, op.to_lowercase());
+            tags.insert(SpanTagKey::TransactionOp, op.to_lowercase().into());
         }
     }
 
     if let Some(transaction_http_status_code) = extract_http_status_code(event) {
-        tags.insert(SpanTagKey::HttpStatusCode, transaction_http_status_code);
+        tags.insert(
+            SpanTagKey::HttpStatusCode,
+            transaction_http_status_code.into(),
+        );
+    }
+
+    if MOBILE_SDKS.contains(&event.sdk_name()) {
+        tags.insert(SpanTagKey::Mobile, true.into());
+    }
```

Comment:
This tag will not be available on metrics, it is just a field in the span data bag to inform metrics extraction.